doc/releases/release-notes-4.1.rst

```diff
@@ -180,6 +180,14 @@ New APIs and options
   * :kconfig:option:`CONFIG_MBEDTLS_PSA_STATIC_KEY_SLOTS`
   * :kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`
 
+* I3C
+
+  * :kconfig:option:`CONFIG_I3C_TARGET_BUFFER_MODE`
+  * :kconfig:option:`CONFIG_I3C_RTIO`
+  * :c:func:`i3c_ibi_hj_response`
+  * :c:func:`i3c_ccc_do_getacccr`
+  * :c:func:`i3c_device_controller_handoff`
```

Comment:
should this also mention support for CONFIG_I3C_DW?

Reply:
That's associated with the driver. The release note does not mention the kconfig associated with newly introduced drivers so I am simply following that.